lib/dpl/provider/lambda.rb

```diff
@@ -197,7 +228,19 @@ def default_kms_key_arn
       def function_tags
         options[:function_tags] ? split_string_array_to_hash(options[:function_tags]) : nil
       end
-
+      
+      def batch_size
+        options[:batch_size] ? options[:batch_size] : 1
+      end
+      
+      def starting_position
+        options[:starting_position] ? options[:starting_position] : nil
+      end
+      
+      def starting_position_timestamp
+        options[:starting_position_timestamp] ? Time.parse(options[:starting_position_timestamp]) : nil
```

Comment:
Are we passing a `Time` object to `#create_event_source_mapping`? How does that work? Also, don't we want to validate that `starting_position` option is set to `AT_TIMESTAMP`?